src/tests/test_config.py

```diff
@@ -146,87 +148,106 @@ def test_magic_initialization_with_corrupted_pyproject(tmp_empty, ip_no_magics,
     assert "Could not load configuration file" in captured.out
 
 
-@pytest.mark.parametrize(
-    "file_content, expect, config_expected",
-    [
-        (
-            """
+def test_loading_valid_pyproject_toml_shows_feedback_and_modifies_config(
+    tmp_empty, ip_no_magics, capsys
+):
+    Path("pyproject.toml").write_text(
+        """
 [tool.jupysql.SqlMagic]
 autocommit = false
 autolimit = 1
 style = "RANDOM"
-""",
-            [
-                "Found pyproject.toml from '%s'",
-                "Settings changed:",
-                r"autocommit\s*\|\s*False",
-                r"autolimit\s*\|\s*1",
-                r"style\s*\|\s*RANDOM",
-            ],
-            {"autocommit": False, "autolimit": 1, "style": "RANDOM"},
-        ),
+"""
+    )
+
+    expect = [
+        "Loading configurations from {path}",
+        "Settings changed:",
+        r"autocommit\s*\|\s*False",
+        r"autolimit\s*\|\s*1",
+        r"style\s*\|\s*RANDOM",
+    ]
+
+    config_expected = {"autocommit": False, "autolimit": 1, "style": "RANDOM"}
+
+    toml_path = str(Path(os.getcwd()).joinpath("pyproject.toml"))
+
+    os.mkdir("sub")
+    os.chdir("sub")
+
+    load_ipython_extension(ip_no_magics)
+    magic = ip_no_magics.find_magic("sql").__self__
+    combined = {**get_default_testing_configs(magic), **config_expected}
+    out, _ = capsys.readouterr()
+    expect[0] = expect[0].format(path=re.escape(toml_path))
+    assert all(re.search(substring, out) for substring in expect)
+    assert get_current_configs(magic) == combined
+
+
+@pytest.mark.parametrize(
+    "file_content, param",
+    [
         (
             """
 [tool.jupysql.SqlMagic]
 """,
-            ["Found pyproject.toml from '%s'"],
-            {},
+            "[tool.jupysql.SqlMagic] present in {path} but empty.",
         ),
+        ("", "Tip: You may define configurations in {path}."),
+    ],
```

Comment:
pytest accepts the `id` argument to idenfity parametrized tests, please add the so it's clearer what scenarios you're testing on each, you can look at existing tests to have an idea

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/823/commits/396e4c8566419b236e817aabe311ee41f6dec137)